### DISCUSSION TITLE
Setting up IDE

### DISCUSSION THREAD

**Discussion by thabaum** [dnnsoftware/Dnn.Platform#3889] [General] [answered]
https://github.com/dnnsoftware/Dnn.Platform/blob/develop/.github/BUILD.md

I am going down this trying to setup my IDE so I can actually build and test some fuctionality I would like to add to the DNN platform.  I must be missing something here as I am trying to follow step by step, word for word how to build, deploy and create a developer environment for DNN platform for both extensions and the platform itself of coarse.

When I build or build to create the settings I get a warning stating to update Cake.Core to version 33 but that is not my problem.

When I build it `.\build.ps1` it gives me 2700 warnings and 1 error with a stop code.  

So then I ran the `.\build.ps1 -Target CreateSettings` and looked around for the file was suppose to create to enter dnn build settings.  Seems like it runs the script ok however no file is created.

Any suggestions?  Thank you.

**Comment by valadas** [comment]
For the warnigns, they are kinda normal for now, we implemented StyleCop to harmonize coding styles and everything not matching those rules comes up as a warning. We are correcting those in batches, some are only whitespace or required comas and mixed single-line vs multi-line, all kinds of things we can automate solution-wide in batches but cause huge PRs. Others will need more of a human intervation like documenting all public members. So long story short, it is currently normal to have that many warnings.

For the error, it is important to spot what it is in order to troubleshoot the issue and maybe it is also the cause of the settings file not generating... Can you post the actual error?

**Comment by thabaum** [comment]
I am not getting an error, I just cannot find a file created after running the command to create the settings file.
![image](https://user-images.githubusercontent.com/13577556/86431269-369d6100-bca9-11ea-8cc2-c14ed8e643f4.png)

**Comment by thabaum** [comment, marked as answer]
would it be because Visual Studio is open while running it possibly?  hah!!! found it after I closed the solution and ran the script command and the file was created.

Not sure maybe it was just me.. but maybe a note to run the commands on the repo folder with powershell while VS is closed.

Thanks for the reply... hoping to get busy here soon!
